src/results/PersonalBenefits.js

```diff
@@ -88,10 +88,10 @@ class PersonalBenefits extends React.Component<Props> {
                   </span>{" "}
                     € / {benefit.periode}
                   </Grid>
-                  <Grid item xs={6} sm={3}>
+                  <Grid className={"Separator"}item xs={6} sm={3}>
```

Comment:
no need to use `{}`

Reply:
As you are not reusing this class anywhere els I'd prefer to have classnames that identify the element like `ResultsPageMoreInfoButtonContainer`

Reply:
space between `className` and `item`. Also `item` should be the first property.